app/code/Magento/Contact/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,50 @@
+// /**
```

Comment:
Hi @suryakant-krish, thanks for collaboration. I think `//` comments tags is redundant